pcdsdevices/tmo_ip1.py

```diff
@@ -0,0 +1,51 @@
+"TMO sample calibration foil to work with spectrometer"
+from lightpath import LightpathState
+from ophyd.device import Component as Cpt
+
+from .device import GroupDevice
+from .device import UpdateComponent as UpCpt
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface, LightpathMixin
+from .pmps import TwinCATStatePMPS
+
+
+class CalibrationAxis(TwinCATStatePMPS):
+    """
+    Sample calibration 1D State Setup
+    Here, we specify 5 states, and 1 motor, for Y
+    axe.
+    """
+    config = UpCpt(state_count=5, motor_count=1)
+
+
+class SCaFoil(BaseInterface, GroupDevice, LightpathMixin):
+    """
+    TMO PMPS1D sample calibration class.
+    Parameters:
+    -----------
+    prefix : str
+        Base PV for the motion system
+    name : str
+        Alias for the device
+    """
+    # UI Representation
+    _icon = 'fa.minus-square'
+    tab_component_names = True
+
+    # Sample calibration Y axe
+
+    pf = Cpt(CalibrationAxis, 'PA1K4:PF:STATE', add_prefix=(), kind='normal')
```

Comment:
You may know this, but by setting `add_prefix=()` here you stop the device prefix from being pre-pended to this component's PV.

Should this also be named something like `state`?  What does `pf` stand for here?

Reply:
thanks Robert. I will push new code then. pf here is for photon filter,  PA1K4 is the location. It is the name committee decision:PA1K4:PF.